diskv.go

```diff
@@ -403,6 +433,9 @@ func (d *Diskv) EraseAll() error {
 	defer d.mu.Unlock()
 	d.cache = make(map[string][]byte)
 	d.cacheSize = 0
+	if d.TempDir != "" {
+		os.RemoveAll(d.TempDir) // errors ignored
```

Comment:
We really shouldn't recommend people to use $TMP, as it must be on the same deviceid as the BaseDir. It's really not uncommon for people to set-up a specific partition for /tmp.